Formula/leapp-cli.rb

```diff
@@ -0,0 +1,27 @@
+require "language/node"
+
+class LeappCli < Formula
+  desc "Cloud credentials manager cli"
+  homepage "https://github.com/noovolari/leapp"
+  url "https://registry.npmjs.org/@noovolari/leapp-cli/-/leapp-cli-0.1.13.tgz"
+  sha256 "944ce4928c2d2dd562890310ea6f2f96d3ace5f1c3b7e8bbc9ef8323403359d8"
+  license "MPL-2.0"
+
+  depends_on "pkg-config" => :build
+  depends_on "python@3.10" => :build
+  depends_on "node"
+  # TODO: fix node-gyp "npm ERR! gyp sill" error
+
+  on_linux do
+    depends_on "libsecret"
+  end
+
+  def install
+    system "npm", "install", *Language::Node.std_npm_install_args(libexec)
+    bin.install_symlink Dir["#{libexec}/bin/*"]
+  end
+
+  test do
+    assert_match "idp-url", shell_output("#{bin}/leapp").strip
```

Comment:
What does this test do?

Reply:
Not much. It's simply checking if the leapp cli is working by checking if it shows the expected cli flag.

Reply:
The issue is that I cannot do a "real" test because the `leapp-cli` also requires the cask `leapp`. I added this in the `caveats`.

Reply:
So you can run a command and see if it fails as expected then?

Reply:
@SMillerDev I'm not sure I'm following.

If I run a leapp cli command like `leapp idp-url create --idpUrl https://example.com", then it will fail unless the `leapp` cask is installed.

Are you saying that there is a way to create a cask dependency for a test ?

Reply:
Hmm oh I think I know what you mean. You're saying I should run a command and ensure it fails. OK I can add this.

Reply:
No, it only runs when someone does `brew test leapp-cli`